elasticdl/python/common/k8s_client.py

```diff
@@ -264,6 +266,14 @@ def _create_pod(self, **kargs):
             args=kargs["container_args"],
             image_pull_policy=kargs["image_pull_policy"],
             env=kargs["env"],
+            ports=[
```

Comment:
These two ports will be added for all pods, including PS which does not use them.

Reply:
Good catch. I've updated the PR to address this. Please take another look.